examples/spring-boot-with-groovy-on-openshift-example/pom.xml

```diff
@@ -1,10 +1,15 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
   <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>io.dekorate</groupId>
+    <artifactId>examples</artifactId>
+    <version>2.12-SNAPSHOT</version>
+  </parent>
```

Comment:
```suggestion
```

Can we please remove this? It shouldn't be needed.

Reply:
It's necessary to update the parent.version when using `mvn versions:set`.

Reply:
PR updated by copying the property `version.groovydoc-maven-plugin` in the pom definition of this example.